### ISSUE TITLE
Bad result of scnn tensorflow on tusimple dataset

### ISSUE THREAD

**Issue by abx67** [cardwing/Codes-for-Lane-Detection#136]
Hi, first really appreciate your works and codes.

I am trying to train SCNN-Tensorflow (based on vgg16.npy). My strategy is to train on a single image and converge quick, then test on same image. I wrote a python script to generate line curve followed by origin Matlab scnn code.

**1. On CULane**
Everything works on CULane data set. I tested at 3000 / 6000 / 9000 epoch and outputs like following:

`Epoch: 3000 loss_ins= 0.005545 (0.005464) loss_ext= 0.000123 (0.000137) accuracy= 0.979704 (0.986540) accuracy_back= 0.996743 (0.996111) mean_time= 0.105044s `
![3000](https://user-images.githubusercontent.com/28716077/63481152-77053f00-c449-11e9-8858-53315a1b7278.png)

`Epoch: 6000 loss_ins= 0.004461 (0.004621) loss_ext= 0.000060 (0.000063) accuracy= 0.988955 (0.988023) accuracy_back= 0.996884 (0.996763) mean_time= 0.102344s `
![6000](https://user-images.githubusercontent.com/28716077/63481189-8f755980-c449-11e9-89fc-c81ef0e58831.png)

`Epoch: 9000 loss_ins= 0.004201 (0.004254) loss_ext= 0.000039 (0.000040) accuracy= 0.990088 (0.988921) accuracy_back= 0.997016 (0.997055) mean_time= 0.105421s 
`
![9000](https://user-images.githubusercontent.com/28716077/63481210-a025cf80-c449-11e9-8b9a-579b055ae3bf.png)

**2. On TuSimple**
But when I do the same on TuSimple dataset. Although accuracy / loss can reach high / low value. But I can not generate a line from output. The final output of training follows like this:
`Epoch: 36346 loss_ins= 0.008268 (0.008306) loss_ext= 0.000015 (0.000015) accuracy= 0.981132 (0.982401) accuracy_back= 0.994675 (0.994569) mean_time= 0.068200s `
`Epoch: 36347 loss_ins= 0.008309 (0.008306) loss_ext= 0.000015 (0.000015) accuracy= 0.985582 (0.982410) accuracy_back= 0.994524 (0.994569) mean_time= 0.068178s `

![20](https://user-images.githubusercontent.com/28716077/63481258-c0ee2500-c449-11e9-95ea-e3dd596a0a03.jpg)

Nothing extracted from instance segmentation output. Even 4 digits of existence_output is around 1 (about 0.9999).

In order to train properly, I changed `__C.TRAIN.IMG_HEIGHT = 256` and `__C.TRAIN.IMG_WIDTH = 512`. And all other codes remain the same.

So if anyone can tell me why i cannot get same result on TuSimple data set?

**Comment by cardwing** [comment]
It is quite weird. But you can visualize the performance of SCNN-Tensorflow in the training set of TuSimple by fusing the predictions with original images. You can also have a look at [ENet-TuSimple-Torch](https://github.com/cardwing/Codes-for-Lane-Detection/tree/master/ENet-TuSimple-Torch) and see if it works.
